services/user-management/src/main/java/com/foodie/user/model/repositories/RoleRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.foodie.user.model.repositories;
+
+// import java.util.Optional;
```

Comment:
You'll probably want to remove this line and others

Reply:
Missed it previously.